app/models/bulkrax/oai_entry.rb

```diff
@@ -54,6 +56,40 @@ def collections_created?
       end
     end
 
+    # To ensure we capture the correct parse data, we first need to establish the factory_class.
+    # @see https://github.com/samvera-labs/bulkrax/issues/702
+    def establish_factory_class
+      # Why send?  Because in some implementations I've found that folks have privatized the
+      # model_field_mappings method.
```

Comment:
What is this comment referring to? I don't see `#send` used anywhere besides the spec

Reply:
Oh, that was there based on needs for Adventist.  But I've made changes there.  I'll amend the comment.